commands/system/sample-color.swift

```diff
@@ -2,15 +2,15 @@
 
 // Required parameters:
 // @raycast.schemaVersion 1
-// @raycast.title Sample Colour
+// @raycast.title Sample Color (Sample Colour)
```

Comment:
```suggestion
// @raycast.title Sample Color
```

I wouldn't complicate it and just keep one version, `color`, as also used [here](https://raycast.com/features):
![image](https://user-images.githubusercontent.com/24907857/126408065-17a80063-4590-4d10-92bd-51ced2c5b2e3.png)